content/dist/styleguide.css

```diff
@@ -814,6 +761,78 @@ time.time-reading, .time.time-reading {
   font-size: 1.55rem;
 }
 
+.socials-form {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+      flex-wrap: wrap
+}
+
+.socials-form li {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  margin: 15px 0;
+  margin: 1.5rem 0;
+  -webkit-box-flex: 1;
+      -ms-flex: 1 1 50%;
+          flex: 1 1 50%;
+}
+
+.socials-form li input,
+    .socials-form li a {
+  margin: 0;
+  -webkit-box-shadow: none;
+          box-shadow: none;
+  border-radius: 0;
+  height: auto;
+  line-height: 17px;
+  line-height: 1.7rem;
```

Comment:
Why both values?

Reply:
open in https://github.com/neomad-team/styleguide/issues/36